R/sequences.R

```diff
@@ -31,15 +35,15 @@
 #'   Computational Extraction, Analysis and Exploitation of Collocations.
 #' @export
 sequences <- function(x, features, valuetype = c("glob", "regex", "fixed"),
-                      case_insensitive = TRUE, count_min = 2, nested=TRUE) {
+                      case_insensitive = TRUE, count_min = 2, len_max= 5, nested=TRUE, ordered=FALSE) {
     UseMethod("sequences")
 }
 
 #' @rdname sequences
 #' @noRd
 #' @export
 sequences.tokens <- function(x, features, valuetype = c("glob", "regex", "fixed"),
-                      case_insensitive = TRUE, count_min = 2, nested=TRUE) {
+                      case_insensitive = TRUE, count_min = 2, len_max= 5, nested=TRUE, ordered=FALSE) {
```

Comment:
for consistency with the min and max arguments in `dfm_select` and `tokens_select` let's change these arguments to `min_count` and `max_length`